src/node.h

```diff
@@ -502,14 +551,40 @@ extern "C" NODE_EXTERN void node_module_register(void* mod);
     }                                                                 \
   }
 
+/* for nodejs built-in modules only */
```

Comment:
This is a very good sign that this, too, would be nicer to have in `node_internals.h` :)

Reply:
@addaleax thanks, actually @bnoordhuis had the same comments. I thought I moved it. I got 2-3 working servers, seems I didn't pick the right one to push the change. let me do it now. Thanks for the comments